packages/salesforcedx-vscode-core/src/commands/util/postconditionCheckers.ts

```diff
@@ -211,7 +212,13 @@ export class TimestampConflictChecker implements PostconditionChecker<string> {
   public async check(
     inputs: ContinueResponse<string> | CancelResponse
   ): Promise<ContinueResponse<string> | CancelResponse> {
-    if (!sfdxCoreSettings.getConflictDetectionEnabled()) {
+    // If the current org is source-tracked, then source tracking
+    // will handle conflict detection.
+    const orgType = await getWorkspaceOrgType();
+    if (
+      orgType === OrgType.SourceTracked ||
+      !sfdxCoreSettings.getConflictDetectionEnabled()
+    ) {
       return inputs;
```

Comment:
So if the user has a source tracked org, but has conflict detection turned on, conflict detection will just quietly get skipped?

Reply:
Gotcha - so the experience for the user would be the same?